std/assembly/math.ts

```diff
@@ -360,7 +371,7 @@ export namespace NativeMath {
     if (w < 0x3FE62E42) {
       if (w < 0x3FF00000 - (26 << 20)) return 1;
       t = expm1(x);
-      return 1 + t * t / (2 * (1 + t));
+      return 1 + t * t / (2 + 2 * t);
```

Comment:
Would rather keep the original for readability, unless this reduces the total number of instructions - does it?

Reply:
FPU/SSE prefer `mul+add` scheme call `madd`. Actually `madd` (FMA) in hardware is usually single operation. See [this](https://en.wikipedia.org/wiki/Multiply%E2%80%93accumulate_operation), and this [answer](https://stackoverflow.com/a/15933677). Even without FMA instruction it usually better for hardware and I hope this could easier for VM git optimizer

Reply:
Ah, didn't know that. Do you know if Binaryen has an optimization for that? Or should it?

Reply:
This usually optimized by `--fast-math` flag in GCC/LLVM so I think binaryen shouldn't do that

Reply:
Maybe, but it's still easier to understand for someone who knows the math behind (or just compares the code do musl to find a bug) it if it remains untouched. Imho we should rather aim to get fused multiply add optimizations into Binaryen instead (if not there yet and there's an actual benefit in VMs actually compiling FMAs), as we really cannot expect a programmer to micro-optimize things like these on their own, and in turn shouldn't have to rely on those as well in stdlib.

Reply:
I could add original line via comment. Is this will be ok?